app/routes/docs.contributing.configuration/route.mdx

```diff
@@ -170,5 +170,12 @@ account for GCN production by registering the email address
       </Description>
       <Default>TNS tooltips disabled</Default>
     </tr>
+    <tr>
+      <Key>`KAFKA_CLIENT_ID`, `KAFKA_CLIENT_SECRET`</Key>
+      <Description>
+        Client credentials for producing GCN Circulars over Kafka
+      </Description>
+      <Default>Kafka producer is disabled</Default>
+    </tr>
```

Comment:
What were the pros and cons of creating dedicated client credentials vs. adding the new scope to the web app's existing app client?

Reply:
Ah so I tried using them, but it threw an error about the credentials already used. Let me run it real quick and I can copy the error here

Reply:
You need to go into the Cognito console, find your app client, and add "Client credentials" to "OAuth 2.0 grant types".

Reply:
I think this warning is telling us that it won't work as we want it to:
![image](https://github.com/nasa-gcn/gcn.nasa.gov/assets/11023698/7c96abe4-3998-4dbe-a9a2-5abf319715e4)

If you select it, it removed the auth code grant

Reply:
Okay, thanks for checking that!